src/app/common/components/modals/dialog-create-or-update-wallet/wallet-form-modal.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="wrapper">
```

Comment:
You should remove this .wrapper div and div below. You should also remove width/height settings for opened modal in `pages-wallets-my-wallets.component.ts`. That way layout will be better. I know it will looks small for now but this form will grow over time.